src/main/java/edu/hm/hafner/analysis/FullTextFingerprint.java

```diff
@@ -38,13 +38,23 @@ public class FullTextFingerprint {
      * Creates a new instance of {@link FullTextFingerprint}.
      */
     public FullTextFingerprint() {
-        this(new FileSystem());
+        this(3);
+    }
+
+    /**
+     * Creates a new instance of {@link FullTextFingerprint}.
+     *  @param contextLines
+     *      *         the number of lines which is used in the fingerprinting process
+     */
+    public FullTextFingerprint(final int contextLines) {
+        this(new FileSystem(), contextLines);
     }
 
     @VisibleForTesting
     @SuppressFBWarnings(value = "WEAK_MESSAGE_DIGEST_MD5", justification = "The fingerprint is just used to track new warnings")
-    FullTextFingerprint(final FileSystem fileSystem) {
+    FullTextFingerprint(final FileSystem fileSystem, final int contextLines) {
```

Comment:
Use the same name for field and parameter:

```suggestion
    FullTextFingerprint(final FileSystem fileSystem, final int linesLookAhead) {
```